docker/Dockerfile.stats-web

```diff
@@ -1,61 +1,46 @@
-FROM node:20 AS base
+FROM node:20-alpine AS base
 
-FROM base AS deps
+ENV NEXT_TELEMETRY_DISABLED 1
+
+FROM base AS development
+
+ENV NODE_ENV development
 
 WORKDIR /app
 
+RUN apk add --no-cache libc6-compat
+
+COPY apps/stats-web ./apps/stats-web
+COPY /packages /app/packages
 COPY package.json /app
 COPY package-lock.json /app
-COPY apps/stats-web/package.json ./apps/stats-web/
 
-RUN apt-get update && apt-get install -y libc6-dev && \
-    ARCH=$(uname -m) && \
-    if [ "$ARCH" = "x86_64" ]; then \
-      npm install @next/swc-linux-x64-gnu; \
-    elif [ "$ARCH" = "aarch64" ]; then \
-      npm install @next/swc-linux-arm64-gnu; \
-    else \
-      echo "Unsupported architecture"; exit 1; \
-    fi
+RUN npm install
 
-RUN npm ci
+CMD ["npm", "run", "dev", "--workspace", "apps/stats-web"]
 
-FROM base AS builder
+FROM development AS builder
 
 WORKDIR /app
 
-COPY --from=deps /app/node_modules ./node_modules
-COPY --from=deps /app/apps/stats-web/node_modules ./apps/stats-web/node_modules
-COPY . .
-
-ENV NEXT_TELEMETRY_DISABLED 1
+ENV NODE_ENV production
 
 RUN npm run build -w apps/stats-web
 
-FROM node:20 AS runner
+FROM base AS production
+
 WORKDIR /app
 
 ENV NODE_ENV production
-ENV NEXT_TELEMETRY_DISABLED 1
-
-RUN addgroup --system --gid 1001 nodejs
-RUN adduser --system --uid 1001 nextjs
```

Comment:
Those user/group comes from the [example dockerfile](https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile). I think so that nextjs run as a non-root user. Do you think it is not needed? If so I guess we should also remove the `--chown=nextjs:nodejs` from the `COPY` commands bellow

Reply:
This is needed for security. If someone potentially hacks into there's less harm that could be done. My reasoning was that it might not be needed for a ui only service as it has no db access. However this can change. Perhaps it's better to get this back. Wdyt?

Reply:
anyway, decided to roll this back and add this to other images too